anvil-python/anvil/commands/haproxy.py

```diff
@@ -34,11 +38,48 @@
 HAPROXY_UNIT_TIMEOUT = (
     1200  # 15 minutes, adding / removing units can take a long time
 )
+HAPROXY_CERTS_DIR = os.path.join(environ['HOME'], "certs/")
+LOG = logging.getLogger(__name__)
+
+
+def validate_cert_file(filepath: str) -> None:
+    if not os.path.isfile(filepath):
+        raise ValueError(f"{filepath} does not exist")
+    with open(filepath, "r") as f:
+        if "BEGIN CERTIFICATE" not in f.read():
+            raise ValueError("Invalid certificate file")
+
+def validate_key_file(filepath: str) -> None:
+    if not os.path.isfile(filepath):
+        raise ValueError(f"{filepath} does not exist")
+    with open(filepath, "r") as f:
+        if "BEGIN PRIVATE KEY" not in f.read():
+            raise ValueError("Invalid key file")
 
 
 class DeployHAProxyApplicationStep(DeployMachineApplicationStep):
     """Deploy HAProxy application using Terraform"""
 
+    _HAPROXY_QUESTIONS = {
+        "ssl_cert": questions.PromptQuestion(
+            "Path to SSL Certificate for HAProxy: ",
```

Comment:
This prompt is ending with `: ` where as the `ssl_key` one does not include those characters. Could you please update the one that needs to be updated? :slightly_smiling_face: